02-technical-guides/virtual-hierarchies/main.ipynb

```diff
@@ -11,53 +11,80 @@
     "tags": []
```

Comment:
<p>I can't remember if we decided that we would be adhering to sentence casing moving forward. If this is the case, I can revert this change.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/atoti/atoti/pull/900/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Confirmed that we are moving forward with sentence casing, thus I removed this change.